### PR TITLE
Update PyLadies avatar list

### PR DESCRIPTION
**Pull request by hugovk**
This PR helps towards points 1, 2 and 4 of https://github.com/pyladies/project-communications/issues/63#issuecomment-667202397:

> Today we set up the automation using Zapier to update the avatar daily, and we also added the step to post a tweet about the new profile picture. Other suggested improvements to this are:
> 
> * add a column with link to the PyLadies chapter website/meetup so we can include it on the tweet
> * add a column with the PyLadies twitter account so we can @-mention them on twitter
> * some logo may need to be resized for twitter
> * add more logos

---

Hello!

I saw your video at https://twitter.com/mariatta/status/1289345727936659457, really good idea to rotate the Twitter avatar! Nice to see my https://hugovk.github.io/python-logos/pyladies/ being of use too :)

This PR updates the CSV to the full list of logos I've so far collected at https://hugovk.github.io/python-logos/pyladies/

The CSV now has columns for the homepage, and for Twitter username.

Note that some don't have a Twitter username listed: either they don't have one or I couldn't find it.

(Perhaps https://hugovk.github.io/python-logos/pyladies/ could even be refactored to load the list of logos from a CSV, then either the Zapier script could read that directly, or it'd make it easier to diff and add new ones to here (and the other direction too).)

### CODE REVIEW

File: pyladies_avatars_rotation.csv [hunk 1]
```diff
@@ -1,16 +1,42 @@
...
+Brazil,Manaus,https://hugovk.github.io/python-logos/pyladies/img/pyladiesmanaus.png,https://www.facebook.com/pyladiesmanaus/,pyladiesmanaus
+Brazil,Natal,https://hugovk.github.io/python-logos/pyladies/img/pyladies_natal.svg,https://pyladiesnatal.github.io/,PyladiesNatal
+Brazil,Paraíba,https://hugovk.github.io/python-logos/pyladies/img/pyladiespb.png,https://www.facebook.com/pyladiespb,pyladiespb
+Brazil,Parnaíba,https://hugovk.github.io/python-logos/pyladies/img/pyLadiesPHB.png,https://pyladiesparnaiba.carrd.co/,pyladiesphb
+Brazil,Porto Alegre,https://hugovk.github.io/python-logos/pyladies/img/pyladies-porto-alegre.png,https://pyladiespoa.pythonanywhere.com/,Pyladiespoa
+Brazil,Recife,https://hugovk.github.io/python-logos/pyladies/img/pyladiesrecife.png,https://linktr.ee/pyladiesrecife,pyladies_recife
+Brazil,Ribeirão Preto,https://hugovk.github.io/python-logos/pyladies/img/Pyladies-RP.png,https://pyladies-rp.github.io/,pyladies_recife
+Brazil,Salvador,https://hugovk.github.io/python-logos/pyladies/img/pyladies-salvador.png,https://pyladiessalvador.github.io/,pyladiessa
```
Comment (Mariatta) [inline pyladies_avatars_rotation.csv:32]:
```suggestion

```

I noticed that the Salvador chapter is using the other logo for their twitter avatar. So let's remove this one and choose the next one.

File: pyladies_avatars_rotation.csv [hunk 2]
```diff
@@ -1,16 +1,42 @@
...
+Brazil,Sorocaba,https://hugovk.github.io/python-logos/pyladies/img/PyLadies Sorocaba.png,https://www.facebook.com/PyLadiesSorocaba,pyladiesorocaba
+Brazil,Sul de Minas Gerais,https://hugovk.github.io/python-logos/pyladies/img/pyladies_sul_de_minas.png,https://www.facebook.com/pyladiesmg/
+Brazil,São Carlos,https://hugovk.github.io/python-logos/pyladies/img/pyladies-saocarlos.png,https://www.facebook.com/PyLadiesSaoCarlos,PyLadiesSanca
+Brazil,São José dos Campos,https://hugovk.github.io/python-logos/pyladies/img/PyLadies São José dos Campos.png,https://www.facebook.com/pyladiesSJC/
+Brazil,São Luís,https://hugovk.github.io/python-logos/pyladies/img/pyladies-sao-luis.svg,https://www.instagram.com/pyladiesslz/,pyladiesslz
+Brazil,Teresina,https://hugovk.github.io/python-logos/pyladies/img/pyladiesteresina.png,https://pyladiesteresina.github.io/,pyladiesthe
+Brazil,Vale Do Paraiba,https://hugovk.github.io/python-logos/pyladies/img/pyladiesvale.png,http://vale.pyladies.com/,pyladiesvale
+Global,Remote,https://hugovk.github.io/python-logos/pyladies/img/remote-logo.svg,https://remote.pyladies.com/,PyLadiesRemote
```
Comment (Mariatta) [inline pyladies_avatars_rotation.csv:42]:
```suggestion
Remote,Remote,https://hugovk.github.io/python-logos/pyladies/img/remote-logo.svg,https://remote.pyladies.com/,PyLadiesRemote
```

We started using "Global" PyLadies as the main PyLadies org, while the Remote PyLadies is still considered a "local" or "over the internet" group. So I think it's best to separate the meaning of "Global" from the "Remote" chapter.

**Comment by hugovk** [comment]
Thanks for the review, updated.

https://github.com/hugovk/project-communications/blob/update-logos-csv/pyladies_avatars_rotation.csv now renders on GitHub.

**Comment by Mariatta** [comment]
Thanks so much!!

**Pull request merged**